src/components/CreateTemplate/index.tsx

```diff
@@ -90,12 +108,19 @@ function CreateTemplate({ isOpen, onClose }: Props) {
       }));
       return;
     }
+    if (!courtDataUrl) {
+      const title = `Fail to get courtDataUrl`;
```

Comment:
Customers may not know what courtDataUrl is, maybe it's better to change it to "Fail to get court image"